crates/matrix-sdk/src/room/timeline/inner.rs

```diff
@@ -682,13 +691,16 @@ async fn handle_remote_event<P: RoomDataProvider>(
     } else {
         Default::default()
     };
+    let is_highlighted =
+        push_actions.iter().any(|a| matches!(a, Action::SetTweak(Tweak::Highlight(true))));
```

Comment:
Maybe we should have a method for this on `Action` in Ruma?

Reply:
You mean an `Action::is_highlight` method?

Reply:
Yeah, exactly.